homeassistant/components/waze_travel_time/__init__.py

```diff
@@ -124,8 +130,8 @@ async def async_get_travel_times(
     avoid_subscription_roads: bool,
     avoid_ferries: bool,
     realtime: bool,
-    incl_filter: str | None = None,
-    excl_filter: str | None = None,
+    incl_filters: Collection[str] = (),
+    excl_filters: Collection[str] = (),
```

Comment:
We should not instantiate mutable objects as a parameter. This means that this instance will be reused instead of a new one every time you call

Reply:
Oh python. Never change. 🤣

Reply:
done